crates/bevy_gltf/src/loader.rs

```diff
@@ -502,7 +502,7 @@ async fn load_gltf<'a, 'b, 'c>(
                 bevy_utils::tracing::debug!(
                     "Automatically calculating missing vertex normals for geometry."
                 );
-                mesh.compute_flat_normals();
+                mesh.compute_normals();
```

Comment:
I'd prefer if this still used `compute_flat_normals()`. It works in both cases and won't change anything about how current meshes are loaded.

Reply:
```suggestion
                mesh.compute_flat_normals();
```

Reply:
My understanding is that the current behavior results in smooth normals being generated for indexed meshes. Do we want to retain this behavior, or switch to computing flat normals for indexed meshes as well?

Reply:
That existing behaviour was added by https://github.com/bevyengine/bevy/pull/11654 but I also didn't think that one should have been merged in that state and is why #12716 was created.

Reply:
And also, that PR is only merged on main, so the behaviour that is released is to either compute flat normals or I believe warn about not being able to compute flat normals on meshes with indices.

Reply:
Ah, got it, the branching strategy went over my head a little bit. Let me revert this change, then.